scripts/build-worker.sh

```diff
@@ -55,5 +59,17 @@ pushd "$(dirname "$0")/../"
             +buildEnvironment "${BUILD_ENVIRONMENT}" \
             +scriptingBackend "${SCRIPTING_BACKEND}" \
             "${BUILD_TARGET_FILTER_ARG}"
+
+        if [[ "${BUILD_TARGET_FILTER}" == "ios" ]]; then
+            echo "--- Building XCode Project :hammer_and_wrench:"
+
+            dotnet run -p "${RUN_UNITY_PATH}" -- \
+                -projectPath "." \
+                -batchmode \
+                -quit \
+                -logfile "${LOG_FILE}" \
+                -executeMethod "Improbable.Gdk.Mobile.iOSLaunchUtils.Build" \
```

Comment:
Why is the `Build` method in `iOSLaunchUtils` 🤔

Reply:
cause you first need to build the XCode project, before you can  launch it. 
Happy to change the name if you have a better suggestion

Reply:
`Improbable.Gdk.Mobile.XcodeUtils.Build` and we can throw all the random stuff we need to hack in for XCode